core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -50,12 +50,16 @@ oppia.factory('explorationData', [
     var explorationData = {
       explorationId: explorationId,
       autosaveChangeList: function(changeList, successCallback, errorCallback) {
```

Comment:
Can you add a comment to explain whether changeList is the incremental changelist (i.e. between draftChangeListId - 1 and the present) or whether it's the full changelist since the last saved/committed version?